src/Utilities/Helpers.js

```diff
@@ -19,6 +19,11 @@ import {
 import { intl } from './IntlProvider';
 import { packageSystemsColumns } from '../SmartComponents/Systems/SystemsListAssets';
 
+export const deleteUnsetObjectValues = (object) => {
```

Comment:
This is very similar to "filterSelectedRowIDs". We should think about consolidating naming of these utils, we have quite a lot of them and I'm not sure if we need all of them